commit b438fb0d1c831cb8c024eb4dc644aeb5114074e6
Author: Christophe Giboudeaux <christophe@krop.fr>
Date:   Sun Jun 3 19:43:34 2018 +0200

    Fix build with Qt 5.11
    
    qt5_use_modules was removed in Qt 5.11. use target_link_libraries instead.

diff --git a/cmake/modules/KstMacros.cmake b/cmake/modules/KstMacros.cmake
index 4ff9c151..bc3d8a6c 100644
--- a/cmake/modules/KstMacros.cmake
+++ b/cmake/modules/KstMacros.cmake
@@ -240,7 +240,7 @@ endmacro()
 macro(kst_link)
 	target_link_libraries(${kst_name} ${ARGV})
 	if(kst_qt5)
-		qt5_use_modules(${kst_name} Widgets Xml Network PrintSupport)
+		target_link_libraries(${kst_name} Qt5::Widgets Qt5::Xml Qt5::Network Qt5::PrintSupport)
 	else()
 		target_link_libraries(${kst_name}
 		${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTXML_LIBRARY} ${QT_QTSVG_LIBRARY} ${QT_QTNETWORK_LIBRARY})
diff --git a/src/libkstapp/view.h b/src/libkstapp/view.h
index 95c8c6a5..2af53938 100644
--- a/src/libkstapp/view.h
+++ b/src/libkstapp/view.h
@@ -14,6 +14,7 @@
 #define VIEW_H
 
 #include <QGraphicsView>
+#include <QMenu>
 
 #include "kst_export.h"
 
